modules/auxiliary/scanner/rdp/cve_2019_0708_bluekeep.rb

```diff
@@ -75,6 +81,18 @@ def run_host(ip)
     status
   end
 
+  def rdp_reachable
+    begin
+      connect
+      disconnect
+      return true
+    rescue Rex::ConnectionRefused
```

Comment:
You can `rescue` directly on the method, since there's so little code in the `begin`.

https://github.com/rubocop-hq/ruby-style-guide#implicit-begin